runtime/lib.rs

```diff
@@ -99,38 +99,44 @@ pub static UNSTABLE_GRANULAR_FLAGS: &[UnstableGranularFlag] = &[
     show_in_help: true,
     id: 7,
   },
+  UnstableGranularFlag {
+    name: "otel",
+    help_text: "Enable unstable OpenTelemetry features",
+    show_in_help: false,
+    id: 8,
+  },
```

Comment:
Could have added this one at the end with id 12 to avoid all other changes 😛

Reply:
no they have to be in alphabetical order, there's a test for it.

Reply:
oh interesting...